py/selenium/webdriver/chromium/webdriver.py

```diff
@@ -35,22 +36,28 @@ class ChromiumDriver(RemoteWebDriver):
     def __init__(self, browser_name, vendor_prefix,
                  port=DEFAULT_PORT, options: Options = None, service_args=None,
                  desired_capabilities=None, service_log_path=DEFAULT_SERVICE_LOG_PATH,
-                 service: Service = None, keep_alive=True):
+                 service: Service = None, keep_alive=DEFAULT_KEEP_ALIVE):
         """
         Creates a new WebDriver instance of the ChromiumDriver.
         Starts the service and then creates new WebDriver instance of ChromiumDriver.
 
         :Args:
-         - browser_name - Browser name used when matching capabilities.
-         - vendor_prefix - Company prefix to apply to vendor-specific WebDriver extension commands.
+         - browser_name - Deprecated: Browser name used when matching capabilities.
```

Comment:
Actually thinking about this we should keep `browser_name` and `vendor_prefix` due to the way inheritance is done.

Reply:
Removed `browser_name` and `vendor_prefix` from deprecation for chromium family drivers for now.